values.yaml

```diff
@@ -23,6 +23,13 @@ httpsEgressSafelist:
 namespaces:
 - name: verify-main
 
+- name: verify-connector-node-metadata
+  owner: alphagov
+  repository: verify-metadata
+  branch: master
+  path: ci/verify
+  requiredApprovalCount: 2
```

Comment:
`requiredApprovalCount: 2`
@philandstuff Can we remove this for now? A lot of Verify devs push to this repo on a weekly basis and only a handful have been added to the [GSP trusted developers repo](https://github.com/alphagov/gds-trusted-developers/tree/master/users)